docs/framework/windows-workflow-foundation/binding-a-custom-activity-property-to-a-designer-control.md

```diff
@@ -3,127 +3,129 @@ title: "Binding a custom activity property to a designer control"
+    return null;
+    ```
+
+     The expression in the above code snippet can also be created using <xref:Microsoft.CSharp.Activities.CSharpValue%601> instead of <xref:Microsoft.VisualBasic.Activities.VisualBasicValue%601>.
+
+    ```csharp
+    ModelItem modelItem = value as ModelItem;
+    if (value != null)
+    {
+        InArgument<string> inArgument = modelItem.GetCurrentValue() as InArgument<string>;
+
+        if (inArgument != null)
+        {
+            Activity<string> expression = inArgument.Expression;
+            CSharpValue<string> csexpression = expression as CSharpValue<string>;
+            Literal<string> literal = expression as Literal<string>;
+
+            if (literal != null)
+            {
+                return "\"" + literal.Value + "\"";
+            }
+            else if (csexpression != null)
+            {
+                return csexpression.ExpressionText;
+            }
+        }
+    }
+    return null;
+    ```
+
+4. Add the following code to the `ConvertBack` method. This code converts the incoming combo box item back to an <xref:System.Activities.InArgument%601>.
+
+    ```csharp
+    // Convert combo box value to InArgument<string>
+                string itemContent = (string)((ComboBoxItem)value).Content;
+                VisualBasicValue<string> vbArgument = new VisualBasicValue<string>(itemContent);
+                InArgument<string> inArgument = new InArgument<string>(vbArgument);
+                return inArgument;
+    ```
+
+     The expression in the above code snippet can also be created using <xref:Microsoft.CSharp.Activities.CSharpValue%601> instead of <xref:Microsoft.VisualBasic.Activities.VisualBasicValue%601>.
+
+    ```csharp
+    // Convert combo box value to InArgument<string>
+                string itemContent = (string)((ComboBoxItem)value).Content;
+                CSharpValue<string> csArgument = new CSharpValue<string>(itemContent);
+                InArgument<string> inArgument = new InArgument<string>(csArgument);
+                return inArgument;
+    ```
+
+## Adding the ComboBoxItemConverter to the custom designer of an activity
+
+1. Add a new item to the project. In the New Item dialog, select the Workflow node and select Activity Designer as the type of the new item. Name the item CustomPropertyDesigner.
+
+2. Add a Combo Box to the new designer. In the Items property, add a couple of items to the combo box, with Content values of "Item1" and 'Item2".
+
+3. Modify the XAML of the combo box to add the new item converter as the item converter to be used for the combo box. The converter is added as a resource in the ActivityDesigner.Resources segment, and specifies the converter in the Converter attribute for the <xref:System.Windows.Controls.ComboBox>. Note that the namespace of the project is specified in the namespaces attributes for the activity designer; if the designer is to be used in a different project, this namespace will need to be changed.
+
+    ```xml
```

Comment:
not sure if this should be `xaml` instead since I saw that on other files. I just usually put `xml` since then syntax highlighting works in VS Code

Reply:
You're right. We use xaml in these cases.

Reply:
```suggestion
    ```xaml
```

Reply:
Ha, was able to suggest my own code change :)

Reply:
It makes it easier right? 😄

Reply:
Yeah, easier to switching to the branch and pulling up the file. Not as clean as a rebase, but here is always squash-merge for that :sparkles: